### ISSUE TITLE
Unable to add mod instance on Totara 12

### ISSUE THREAD

**Issue by logan-reynolds** [netspotau/moodle-mod_lightboxgallery#87]
Got the following error which prevents me from adding an instance on Totara 12 using the latest M33 version of mod_lightboxgallery version 2018062700:
```
Exception - Class 'core_completion\api' not found
Debug info:
Error code: generalexceptionmessage
Stack trace:

    line 84 of /mod/lightboxgallery/lib.php: Error thrown
    line 121 of /course/modlib.php: call to lightboxgallery_add_instance()
    line 176 of /course/modedit.php: call to add_moduleinfo()
```

**Comment by aolley** [comment]
Hi - I've just tested this on the latest version of Moodle and dev versions (3.6 and 3.7+) where this continues to work as expected.

I'm not sure what version of Moodle T12 is based on - but either way I think you'll be better served with filing a support request with Totara.

That class should be autoloaded automatically, and has been available in Moodle since version 3.3.

**Comment by logan-reynolds** [comment]
Found this is caused by Totara 12 omitting the Activity Completion API ( /completion/classes/api.php file ) - closing
